Makefile

```
@@ -12,10 +12,12 @@ ifeq ($(OS), darwin)
 OPENSSL_DIR ?= $(shell brew --prefix)/opt/openssl
 GRPCURL_OS ?= osx
 YAML_DIR ?= $(shell brew --prefix)/opt/libyaml
+CURL_INCDIR ?= $(shell brew --prefix)/opt/curl/include/curl
```

My bad I tested this, and this seems to be correct:
```suggestion
CURL_INCDIR ?= $(shell brew --prefix)/opt/curl/include/
```